ui/src/app/pages/profile/profile-styles.ts

```diff
@@ -21,6 +21,9 @@ export const styles = reactStyles({
     fontSize: 14,
     marginRight: 20,
   },
+  disabledInput: {
+    backgroundColor: colorWithWhiteness(colors.disabled, 0.6),
```

Comment:
right now , the disabled text boxes are were very hard to distinguish added a little more grey to make it obvious